webapp-django/crashstats/settings/base.py

```diff
@@ -34,6 +34,8 @@ def path(*dirs):
     cast=bool
 )
 
+# Whether or not we're running in the local development environment
+LOCAL_DEV_ENV = config('LOCAL_DEV_ENV', False, cast=bool)
```

Comment:
We don't want to (ab)use `DEBUG` to denote whether we're in a local development environment because we want to be able to flip that on and off to test other things. We really need a separate variable.

I'm game for changing the name. I'm not hung up on this one.